.travis.yml

```diff
@@ -13,10 +13,23 @@ before_install:
   - carthage bootstrap --use-submodules --no-build
 
 script:
+  - carthage build --no-skip-current
+  - carthage archive APIKit
   - pod spec lint APIKit.podspec
   - set -o pipefail
   - xcodebuild test -workspace APIKit.xcworkspace -scheme APIKit-iOS -sdk iphonesimulator | xcpretty -c
   - xcodebuild test -workspace APIKit.xcworkspace -scheme APIKit-Mac | xcpretty -c
 
+deploy:
+  provider: releases
+  api_key:
+    secure: I/cUbYW2BO/Yej52SHoHEmsjhP27jZuQl7tTvjliNzDAmEeYp6HR4fMnxUUUqcntaqrk6ZJauQ0GjbcDqkekPUAZAaUCsT5kh0qqGK8yMpSTd+FhBHNjqOvPimwGqKn9HOmGIJQnH8LqG+1esZhvhQEJJ0rpRSvapETUIcrZewI=
+  file: APIKit.framework.zip
+  skip_cleanup: true
+  on:
+    repo: ishkawa/APIKit
+    tags: true
+    all_branches: true
```

Comment:
Since `tags` is set, this [would be ignored](http://docs.travis-ci.com/user/deployment/#Conditional-Releases-with-on%3A). So it should be removed, I think.